docs/topics/maven-logging-properties.adoc

```diff
@@ -2,7 +2,7 @@
 //
 // * docs/maven-guide/master.adoc
 
-[id='maven-logging-properties_{context}']
+[id="maven-logging-properties_{context}"]
 = Default `logging.properties` in the {MavenNameTitle}
```

Comment:
OpenShift docs says not to use backticks in headers. Sometimes they render strangely. Also, the name does not look good. Suggest you change it to either "Default logging properties" or "Default logging.properties file". I think the former is better.